.buildkite/commands/prototype-build-wordpress.sh

```diff
@@ -14,4 +14,4 @@ echo "--- :closed_lock_with_key: Installing Secrets"
 bundle exec fastlane run configure_apply
 
 echo "--- :hammer_and_wrench: Building"
-bundle exec fastlane build_and_upload_jetpack_installable_build
+bundle exec fastlane build_and_upload_wordpress_prototype_build
```

Comment:
I love how git got confused about the renaming and ended up producing this diff.